tools/android_lint/bin/main.dart

```diff
@@ -6,8 +6,8 @@
 
 import 'sound_main.dart' as m;
 
-// TODO(#72542):
-// Migrate the deps in pubspec.yaml to null-safety versions.
+// TODO(dnfield): Migrate the deps in pubspec.yaml to null-safety versions.
 // In particular see the ongoing work on package:args here:
-// https://github.com/dart-lang/args/issues/153
+// https://github.com/dart-lang/args/issues/153,
```

Comment:
I'm pretty sure we can remove this line since args has been migrated and we'll be getting the migrated package now.

Reply:
Done